README.md

```diff
@@ -58,7 +58,7 @@ We recommend putting Remit inside [Fluid.app](https://fluidapp.com/) or equivale
 
 Fluid.app is macOS only. Please do contribute instructions for other platforms.
 
-* Install [Fluid.app](http://fluidapp.com/). You can use Homebrew: `brew cask install fluid`
+* Install [Fluid.app](http://fluidapp.com/). You can use Homebrew: `brew install fluid`
```

Comment:
(Minor: A tiny detail would be to use `install --cask`, which mentions the "caskness" in another way.)

Reply:
Is the fact that fluid is installed as a cask important in this case? (I honestly don't know.)

Reply:
My guess: it's a way to "assist" the resolution mechanism in brew.

Reply:
`--cask`? Yes, according to the docs, it instructs brew to "Treat all named arguments as casks". I'm not sure it's needed in this case, though? `brew install fluid` **does** install Fluid as a cask, but is it important that it is installed as a cask and not a formula?

Reply:
To be clear: Your change is better than my _Minor_ note and my guess.